paper/mel_report.Rmd

```diff
@@ -31,37 +34,237 @@ knitr::opts_chunk$set(fig.align = 'center',
                       dpi = 300)
 
 ## SOURCE FIGURES AND TABLES
+source(paste(disc_project_wd, "/code/participant_id_cleaning.R", sep = ""))
 source(paste(disc_project_wd, "/code/table_1.R", sep = ""))
 source(paste(disc_project_wd, "/code/mel_report_analysis.R", sep = ""))
 source(paste(disc_project_wd, "/code/figures.R", sep = ""))
+source(paste(disc_project_wd, "/code/table_scores.R", sep = ""))
+source(paste(disc_project_wd, "/code/indicator_table.R", sep = ""))
+```
+
+# Background
+As a part of the three-year USAID-funded Harmonia Activity, HAMNASA partnered with the Ministry of Health, Ministry of Social Solidarity and Inclusion, and the National Institute of Health to address gender-based violence (GBV) in Timor-Leste. The Harmonia activity focused on two municipalities within Timor-Leste, Ermera and Liquica, both of which experience high rates of GBV. The Harmonia activity had two components: a community microplanning intervention and implementation of a 'Responding to GBV' curriculum for healthcare providers. This report aims to evaluate the overall impact of the 'Responding to GBV' curriculum. 
+
+The 'Responding to GBV' curriculum's goal was to engage healthcare providers to shift social norms and protect victims of violence. The curriculum was based upon the WHO curriculum for training healthcare providers caring for women subjected to GBV and its previous adaptation for nursing, midwifery and medical degrees in Timor-Leste.
+
+The curriculum included an initial five days of intensive training, followed by monthly ‘learning lab’ sessions to be carried out at health facilities over a 14-month period. In an effort to create a health-system wide response to GBV, participants included medical doctors or nurses, as well as health facility managers, pharmacists, lab analysts, and other positions. From July to November 2021, eleven intensive trainings took place at health facilities in Ermera and Liquica, training a total of 232 healthcare workers on GBV. These five-day intensive trainings consisted of 14 two-hour modules on various gender-based violence topics. Learning labs, which were shorter 2-hour trainings covering one GBV module each, ran for 14-months following the intensive trainings, from March 2022 to June 2023.
+
+## Monitoring, Evaluation, and Learning Targets
+Three indicators included in the Harmonia Activity’s monitoring, evaluation, and learning plan will be used to assess the success of the ‘Responding to GBV Learning Labs’ Curriculum (Table 1). Two are direct measures of the success of the ‘Responding to GBV Learning Labs’ Curriculum (outcomes 4 and 5). The third measure (outcome 7) will be reported as a complementary finding to the official monitoring, evaluation, and learning plan’s use of health facility data to identify change in the number of clients being identified as victims of GBV.
+
+```{r}
+indicator_table
```

Comment:
You can't actually use this because it is HTML output. You have two options: change this to be compatible with a word doc, or just use the rmd that I had originally included that will produce a table in word: 

`| Result                                                                                                                | Indicator                                                                               | Frequency of Collection                                                                                                   | Unit of Measure                                                                                                                                         | Value          |
|-----------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
| Outcome 4: Health providers who have participated in the ‘Responding to GBV Learning Lab’ Curriculum demonstrate improved knowledge related to GBV | Indicator 4: % of health facilities demonstrate improvement in staff knowledge, attitude, and empathy in responding to GBV | Three time points: pre-training, post-intensive training, and post-intervention                                             | Average composite score of knowledge, attitude, and empathy domains by health facility cohort pre- to post-intervention | 80% of health facilities |
| Outcome 5: Health providers who have participated in the ‘Responding to GBV Learning Lab’ have increased confidence in delivering survivor-centered care   | Indicator 5: % of health facilities that demonstrate improvement in health provider confidence in responding to GBV | Three time points: pre-training, post-intensive training, and post-intervention                                             | Average composite score of the confidence, system support, and professional role domains  by health facility cohort pre to post-intervention | 80% of health facilities |
| Outcome 7: Increase in clients identified who are victims of GBV                                                     | Indicator 7: % increase in GBV clients identified                                         | Two timepoints: pre-training and post-intervention* *This measure can be used to compare to health facility reporting (which is the official MEL data source) on clients identified | Percentage compared to baseline | 50% increase   |`